suite_cam7.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="dadadj" version="1.0">
```

Comment:
Whoops, forgot to mention in my previous review that this should probably be changed to `cam7` as well:

```suggestion
<suite name="cam7" version="1.0">
```